Command/ImportCommand.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace EMS\CoreBundle\Command;
+
+use Elasticsearch\Client;
+use EMS\CoreBundle\Entity\ContentType;
+use EMS\CoreBundle\Exception\CantBeFinalizedException;
+use EMS\CoreBundle\Exception\NotLockedException;
+use EMS\CoreBundle\Service\ContentTypeService;
+use EMS\CoreBundle\Service\DataService;
+use EMS\CoreBundle\Service\ImportService;
+use Monolog\Logger;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use ZipArchive;
+
+class ImportCommand extends EmsCommand
+{
+    /** @var ImportService */
+    private $importService;
+    /** @var ContentTypeService */
+    private $contentTypeService;
+    /** @var DataService */
+    private $dataService;
+
+    public function __construct(Logger $logger, Client $client, ContentTypeService $contentTypeService, ImportService $importService, DataService $dataService)
+    {
+        $this->contentTypeService = $contentTypeService;
+        $this->importService = $importService;
+        $this->dataService = $dataService;
+        parent::__construct($logger, $client);
+    }
+    
+    protected function configure()
+    {
+        $this
+            ->setName('ems:document:import')
+            ->setDescription('Import a bunch of json files from a zip file')
```

Comment:
I have updated the description 866a1ca69e8a617aacadf3a89d5957812f4a0f0e. To me the maker bundle is something different. This is an import/export core functionality. It can be refactored later within the maker bundle. But I need this feature soon.